online-shop/src/app/interfaces/ProductInCart.ts

```diff
@@ -0,0 +1,6 @@
+//import {Product} from "./Product";
+
+export interface ProductInCart {
+  product: any,
```

Comment:
Please avoid using any, especially when you have a class/interface already existing for that type.